### ISSUE TITLE
Proposal for next steps

### ISSUE THREAD

**Issue by bradreeder** [InFact-coop/InFact#30]
### Why

I think there's a lot of confusion at the moment about what people are committing to and how to proceed in the short-term. The following is only my suggestion.

### Requirements

This proposal hinges on us finding enough work to support all of our members financially for the period of time they commit to. If we can secure the Anna Freud project #21  and the NHS Re-think project #22 , and the work can be completed between the six of us, this should be more than enough for the first few months.

### What

A requirement for taking on this project work with the agency is that those involved should accept the terms of a provisional membership up until the project finishes. My initial suggestion would be:

+ you agree to work full-time towards the advancement of the agency.
+ all of the money earned from these projects will be divided evenly among those who accept the agreement.

If someone is working on projects full-time that counts towards their time. Anything else is voluntary and not expected, beyond attending meetings & stand-ups. 

If, however, you're not working on a project then you're expected to do other work to advance the agency during these times. This could include anything from networking / sales, to admin, to researching / making proposals for organisational things, to applying for training opportunities, to working on the website, etc. Work that by consensus we agree provides value.

We should have regular stand-ups in which everyone reports on what they have done and will be doing to advance the agency, so that everyone is accountable to each other. If people do not keep to the agreement their provisional membership can be revoked.

### Proposed benefits

* If people are unsure they want to commit to anything long-term, they're only committing to a period of a few months. If they're not happy with how things are progressing, they can leave once the work is done. There's no hard feelings because their work supported everyone to advance the agency at the time they participated.
* The agency agreement is such that everyone who commits is financially secure for the period they commit.
* Progress of the agency is guaranteed because this means people are being paid to do things that would otherwise be voluntary when they're not working on projects. If the agency disbands, at least everyone was compensated & has a portfolio piece to show. If people leave and others want to continue the agency, they will have more of a foundation to work with than before.
* We have collective responsibility & accountability to each other, which is necessary if we're going to build this as co-founders.
* Those not on projects will need to do things they might not want to. However, we need to get used to wearing different hats initially if we really want to build an actual organisation together. At least jobs that would otherwise be voluntary get done at times in which someone gets compensated for it.

### Potential pitfalls

* I am unsure what everyone thinks of an equal pay-structure. It is a departure from @sofer 's proposal #15 , but not uncommon (or mandatory) in co-ops. We could review if we think this works after the first few months.
* We need to make sure income from projects is always coming in in such a way that everyone is secure.
* People are not compensated for any extra hours they put in outside the agreement. But they're not expected to put in extra either.

NOTE: Alternative is to compensate people on an hourly rate and give everyone the same hourly rate. I think full-time is better as it's a sign of an actual commitment for a defined period of time. Building an organisation takes work. If we do things wishy-washy it won't happen.

**Comment by bradreeder** [comment]
Basically I'm proposing this as a research phase, testing the waters of our relationships to each other during these projects in a safe way. We're all secure for the period of time we commit. We won't create an official legal structure in this time and can decide on that once the period is finished, but we will have a stronger idea of what this would all look like by the end of it.

We could run a number of organised brain-storming meetings in this time around questions of values, principles, mission, vision etc. They don't need to be decided in advance in order to progress with the agreement, the only thing that is needed is enough work to support everyone.

**Comment by jsms90** [comment]
Completely agreed. I am in full support.

As @sofer has repeatedly said, the key is commitment. I think dividing the money is the fairest way of rewarding _everyone's_ commitment, not just the developers'. I would be concerned about how well we could support all of our members, if we paid the developers much more than everyone else.

I'm also definitely in favour of the full time idea. Again, this is about commitment. Pay based on hours might work down the line. But considering the amount of work involved in setting up a business from scratch, I can't see this getting off the ground if we only have a couple of us making a full time commitment.

Also, starting by compensating everyone equally sets the tone for how much we value each role within the organisation. Beside that, it's much harder to take money off people's salaries to redistribute it, than it is to jointly agree to pay someone more when it started as equal.

Plus, I think we will be freer to find the roles that suit us and figure out which ones the agency truly needs. Especially if we get together regularly to justify the time spent.

On that note, one thing I will say is that this kind of model could easily cause resentment. If we're going to make sure that none of this arises, and everyone agrees that time is being spent on the right things, then regular progress reports _definitely_ seem like the way to go.

...If we're going to make Agile analogies though :stuck_out_tongue_winking_eye:, it might be better to think of this as sprint planning and/or a retrospective (rather than a stand up). Stand ups are supposed to be short and punchy, where each person only speaks for a few minutes. But I really think that these will be crucial in keeping the team together, so it would probably be more realistic to expect these to take longer than a stand up. :upside_down_face:

**Comment by des-des** [comment]
I guess for me, this is not so good. For the foreseeable future i will not be available full time, since i have other FAC commitments.

**Comment by des-des** [comment]
To clarify, i think this sounds great + makes lots of sense, it just would not work for me right now.

**Comment by des-des** [comment]
Also, i think a good exercise at this point would be to think about how this would work logistically.

Eg, Berkeley homes had a 6k budget, and required 24 Dev days, how would everyone's day rates work out? What is our ratio of Devs to non-devs?

**Comment by emilyb7** [comment]
In principle, I think this is really good and seems very much in the spirit of being a cooperative.

Practically speaking, here are some questions that come to mind:

**Financial viability**
- How much budget are the two projects that we have lined up bringing in? And what's the timescale? If we combine those two budgets and divided them out across 6 team members / x number of months, what income does that give us? Are we all happy with that income?
- I think this is extremely important. Honestly, I have no idea how much money I can expect from this and my own commitment depends on it.

**Time commitments**
- We all have other commitments, some more so that others. I'm probably able to commit to full-time work, but I'm not sure this would work for everyone.
- We're all very involved in FAC, and working with FAC is kind of part of our role at the agency. But how much work for FAC can be considered as part of our remit, and how much is voluntary and up to the individual?
- How many hours constitutes a full-time week? Perhaps we can go for slightly less than full-time, e.g. 30 hours. 
- And I think realistically (based on how many things I know you lot are working on) some people will have to do less than full time. Someone working 20 hours can then take 2/3 of what someone doing 30 hours is taking.
- In the interests of keeping things fair, each person would then probably have to stick to that commitment (in terms of the number of hours) we've been paid for the first couple of projects.
- If someone has other sources of income (e.g. Sohil is working for Dwyl), is it not better for all of us that he hold on to that source of income for the time being rather than committing to full time? It's less risk for that person, and means that (by taking on less hours at the agency) there is more budget for those who need it / have more time to spare.

In short, I think we need to consider how many hours each team member can commit to, work out a projected income from projects, and do some actual calculations.

**Comment by bradreeder** [comment]
@emilyb7 @des-des 

* Realistically, I think we have to acknowledge that building a start-up, at least in the building phase, will not be as lucrative as finding a job or working as a free-lancer. I agree the requirement is that the money coming in divided by 6 is enough to support each member for the period of time they participate. 

* According to this agreement, one member wouldn't be putting in less hours than another member. What I think we absolutely need, if this is to proceed, is to make sure that non-project work, that is necessary to the building of this, is not all being done voluntarily (and is getting done). If Noga, for example, was not involved in a project for a week and decided to design the website + logo, that is work that provides value, and she should be getting paid for it. The amount that people do at FAC for free is unreasonable, and I don't want it to carry on into what is a commercial enterprise.

* We could discuss part-time options or basing it on an equal hourly rate? But I think we need a minimum number of people doing full-time for this to be viable, otherwise it will always be a side-project and won't build momentum. I think we need to bare in mind that FAC really needs an agency to get built-up to anticipate when dwyl leaves. Building this is time that provides immense value to FAC. I wouldn't be against considering full-time 30 hours though, and say that we donate back to FAC through our time rather than money initially.

> In short, I think we need to consider how many hours each team member can commit to, work out a projected income from projects, and do some actual calculations.

Sure, sounds good.

**Comment by des-des** [comment]
@bradreeder were you going to breakdown the logistics of the proposal?

**Comment by jsms90** [comment]
Perhaps this is more realistic, given people's other commitments?
> Alternative is to compensate people on an hourly rate and give everyone the same hourly rate.

In which case, maybe an easier way of going about this would be to ask how many hours per week each person feels that they could commit?

**Comment by des-des** [comment]
### My situation

I do not want to be contrary, but the amount of time I could commit to this would be dependent on how rates turned out. I seem to have been averaging at doing 2 days of paid work a week, which has been enough to support me in organising in Nazareth, but would not be possible on a lower rate.

In the scenario that I was not managing stuff in Naz and had more time obviously this would be different and I would have more time.

I totally understand the motivation behind this proposal, and am not trying to disagree. I just think the logical next step, is to start doing some basic calculations. Since I cannot commit 2 days a week at £50 per day.. I think this exercise would be useful to everyone.

### Can we think through the implications?

Eg, can we answer the following question.

Given a project Budget, how would we split budget between work on that project, and work internal to the coop.

To get the ball rolling ill try and do a rough breakdown of how my last project would have worked.
28 dev days + 5 days design, + 4 days project management (these numbers could be wrong, and I am including days that we went over our estimates @sohilpandya ?)

So this is 37 days work required for budget of £5400 (£145pday).

 + Case 1: We would put aside 10 percent of the budget for the coop. In this case we all receive £132 per day.
 + Case 2: We would put aside 50 percent of the budget for the coop. In this case we all receive £97 per day.

I think it would be really useful, if @bradreeder and @jsms90 could also make some estimates based on projects you have worked on, we can start to get a better idea of how this proposal might work in practice, and, more importantly, start to work out what sort of extra work we can support through project work.

I think another useful thing would be if everybody posted 2 numbers: a, the minimum day rate the could survive on and b, the day rate they would like.

### What are the dangers of not tying developer pay to completion of a project?

Finally (sorry), I think there is one more aspect of this that is important. Historically, work in founders and coders has been paid on delivery of a project, paid in units of work, not time (The developers use a day rate and time estimates to set this price, but if it takes more or less time, the pay is the same). What we are suggesting here is fundamentally a move away from this. If non developers are paid by the day / hour, it would be unfair to say that the developers do not get the same security. Do we feel comfortable doing this? I think, normally, junior developers under estimate. In the case where developers were paid per day, we would go over budget on a project.. This seems dangerous and scary.
